ignite/contrib/handlers/tqdm_logger.py

```diff
@@ -49,7 +49,10 @@ def __call__(self, engine, logger, event_name):
         if logger.pbar is None:
             logger._reset(pbar_total=pbar_total)
 
-        desc = self.tag
+        max_epochs = engine.state.max_epochs
+        default_desc = "Iteration" if max_epochs == 1 else "Epoch"
+
+        desc = self.tag or default_desc
```

Comment:
@g-karthik None should be handled here. Are we missed something ?

Reply:
@vfdev-5 oh gotcha, the `or` condition should handle it, makes sense!